### PR TITLE
productsPage tests

### PR DESCRIPTION
**Pull request by LiviuAC**

### CODE REVIEW

File: test/specs/productDetailed.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        afterEach(async function () {
+            await ProductsPage.logout();
+        });
+
+        it("should display only the 'Sauce Labs Backpack' details upon clicking on its image", async () => {
+            await ProductsPage.allItemsImages[0].click()
+
+            expect(await ProductDetailedPage.detailsImage.isDisplayed()).toBe(true);
+            expect(await ProductDetailedPage.detailsImage.getAttribute('src')).toEqual(ImageSource.backpack);
```
Comment (axboory) [inline test/specs/productDetailed.js]: It's just checking for the image. It should also check for item name, description, price, add to cart button.

File: test/specs/products.js [hunk 2]
```diff
@@ -22,22 +23,32 @@ describe('Products Page Tests', () => {
...
-        it('should have the filter dropdown menu in header', async () => {
+        it("should have the 'PRODUCTS' text in secondary header", async () => {
+
+            expect(await InventoryPage.productsHeader.getText()).toEqual("PRODUCTS");
+        });
+
+        it('should have the Robot Peek in secondary header', async () => {
+
+            expect(await InventoryPage.robotPeek.isDisplayed()).toBe(true);
+        });
```
Comment (axboory) [inline test/specs/products.js]: You should combine these tests into one test since they are only expects.

File: wdio.conf.js [hunk 3]
```diff
@@ -34,6 +34,9 @@ exports.config = {
         ],
         products: [
             './test/specs/products.js'
+        ],
+        productDetailed: [
+            './test/specs/productDetailed.js'
```
Comment (axboory) [inline wdio.conf.js]: why productDetailed and not productDetails?

**Review by axboory — CHANGES_REQUESTED** [review]

File: test/specs/productDetailed.js (hunk 1 above)
Comment (LiviuAC) [reply test/specs/productDetailed.js]: the TCs for the productDetails are still under development. the checks for the item name, description, price and the add to cart button will be added

File: test/specs/products.js (hunk 2 above)
Comment (LiviuAC) [reply test/specs/products.js]: the header and the footer tests have been grouped under the "check Header UI elements", respectively ""check Footer UI elements" test cases

File: wdio.conf.js (hunk 3 above)
Comment (LiviuAC) [reply wdio.conf.js]: correction made

**Review by axboory — APPROVED** [review]

**Pull request merged**